util/src/main/java/org/eclipse/rdf4j/common/io/ZipUtil.java

```diff
@@ -45,9 +50,7 @@ public static boolean isZipStream(InputStream in)
 	 * @throws IOException
 	 *         when something untoward happens during the extraction process
 	 */
-	public static void extract(File zipFile, File destDir)
-		throws IOException
-	{
+	public static void extract(File zipFile, File destDir) throws IOException {
```

Comment:
Unrelated to the security fix. It seems you’re not following the style guide. Mind you, I think the style guide is old and terrible, designed for the screens we had in the 1980s.

Reply:
Thanks, I'll fix that. Probably best to switch to Eclipse IDE after all, as I'm heavily influenced by the more compact Sun Java code conventions (and Netbeans + the Eclipse-formatter don't seem to work as expected)